code/modules/core_implant/cruciform/rituals/crusader.dm

```diff
@@ -54,7 +54,7 @@
 	phrase = "Per fidem enim ambulamus et non per speciem."
 	desc = "Knocks over everybody without cruciform in the view range. Psy-wave is too powerful, speaker can be knocked too."
 	cooldown = TRUE
-	cooldown_time = 10 SECONDS
+	cooldown_time = 2 Minutes
```

Comment:
Minutes has to be in all capitals, as it's a define

Reply:
I can't believe I did that.

Reply:
I can't believe I did that.